src/backend/lib/upgrade-download.js

```diff
@@ -164,6 +164,7 @@ class Search extends EventEmitter {
 
       this.discovery.destroy(err => {
         if (err) {
+          // Ignorable error
```

Comment:
I think this means that the Search state is in `IDLE` right, but there was an error event?

Reply:
Right now the components goes into its permanent `"Error"` state. There are no `"Error"` events in the upgrade system.